rubocop.yml

```diff
@@ -185,9 +190,19 @@ Lint/IncompatibleIoSelectWithFiberScheduler:
 Lint/InterpolationCheck:
   Enabled: false
 
+<% if rubocop_version >= v1_59 %>
+Lint/ItWithoutArgumentsInBlock:
+  Enabled: true
+<% end %>
```

Comment:
This takes the same approach as #582, and conditionally enables the cop based on the RuboCop version, allowing us to keep the minimum version the same.

Since I realized we'd end up repeating `Gem::Version.new(...)` all over the place, I extracted `v1.xx` variables.